packages/fiori-freestyle-writer/templates/listdetail/add/webapp/controller/ListSelector.ts

```diff
@@ -19,12 +19,12 @@ export default class ListSelector extends UI5Object {
      */
      constructor() {
         super();
-        this._oWhenListHasBeenSet = new Promise(function (this: ListSelector, fnResolveListHasBeenSet: ListSelector['_fnResolveListHasBeenSet']) {
+        this._oWhenListHasBeenSet = new Promise(function (this: ListSelector, fnResolveListHasBeenSet: ListSelector["_fnResolveListHasBeenSet"]) {
             this._fnResolveListHasBeenSet = fnResolveListHasBeenSet;
         }.bind(this));
         // This promise needs to be created in the constructor, since it is allowed to
         // invoke selectItem functions before calling setBoundList
-        this.oWhenListLoadingIsDone = new Promise(function (this: ListSelector, resolve: Function, reject: Function) {
+        this.oWhenListLoadingIsDone = new Promise(function (this: ListSelector, resolve: any, reject: any) {
```

Comment:
This does not look like an improvement to me.

Reply:
Do you need to explicitly type resolve and reject? If you type `oWhenListLoadingIsDone` better with something like `Promise<{ oList: List}` and then say `new Promise<{ oList: List}(function (this: ListSelector, resolve, reject) {});` then both functions should implicitly be typed. (I think :) )

Reply:
@devinea did you try my suggestion?

Reply:
I tried this suggestion, but it doesnt seem to work from my testing. While the lint check passes without error, the typescript build throws the error `TS7006: Parameter 'resolve' implicitly has an 'any' type.` for the removed types.

I just noticed that VSCode offers a 'quick fix' for the missing type, this is what is generated:
```
new Promise(function (this: ListSelector, resolve: (arg0: { oList: List; }) => void, reject: (arg0: { oList: List; }) => void) {
...
}
```
Lint and build steps are ok with this, and the app loads correctly. Thoughts?